### PR TITLE
docs: 📝 Include migration steps for Fleek.co's SDK

### PR DESCRIPTION
**Pull request by punkbit**
## Why?

Include migration steps for Fleek.co's SDK.

⚠️ Related to fleek.co's doc https://github.com/FleekHQ/fleek-docs/pull/167

## How?

- Update SDK Overview page, including the migration steps

## Tickets?

- [PLAT-1484](https://linear.app/fleekxyz/issue/PLAT-1484/docs-for-migrating-fleekcos-sdk)

## Contribution checklist?

- [x] The commit messages are detailed
- [x] The `build` command runs locally
- [ ] Assets or static content are linked and stored in the project
- [ ] Document filename is named after the slug
- [ ] You've reviewed spelling using a grammar checker
- [ ] For documentation, guides or references, you've tested the commands and steps
- [ ] You've done enough research before writing

## Security checklist?

- [ ] Sensitive data has been identified and is being protected properly
- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The Components are escaping output (to prevent XSS)

## References?

Optionally, provide references such as links

## Preview?

Screenshot 2024-09-17 at 11 11 01

### CODE REVIEW

File: src/content/docs/SDK/index.md [hunk 1]
```diff
@@ -129,3 +133,99 @@ const fleekSdk = new FleekSdk({
...
+
+```js
+# New syntax
+import { FleekSdk, ApplicationAccessTokenService } from '@fleek-platform/sdk';
+
+# Using the Application Token
+# Learn more about the available token services
+# by following the link below
+const applicationService = new ApplicationAccessTokenService({
+  clientId: <APPLICATION-CLIENT-ID>,
```
Comment (RobertPechaCZ) [inline src/content/docs/SDK/index.md:165]: missing projectId

File: src/content/docs/SDK/index.md [hunk 2]
```diff
@@ -129,3 +133,99 @@ const fleekSdk = new FleekSdk({
...
+
+```js
+# New syntax
+import { FleekSdk, ApplicationAccessTokenService } from '@fleek-platform/sdk';
+
+# Using the Application token service
+# Learn more about the available token services
+# by following the link below
+const applicationService = new ApplicationAccessTokenService({
+  clientId: <APPLICATION-CLIENT-ID>,
```
Comment (RobertPechaCZ) [inline src/content/docs/SDK/index.md:200]: missing projectId ?

File: src/content/docs/SDK/index.md (hunk 1 above)
Comment (heldrida) [reply src/content/docs/SDK/index.md:165]: @RobertPechaCZ That'd be for `PersonalAccessTokenService`.

File: src/content/docs/SDK/index.md (hunk 2 above)
Comment (heldrida) [reply src/content/docs/SDK/index.md:200]: @RobertPechaCZ  That'd be for PersonalAccessTokenService.

**Review by RobertPechaCZ — APPROVED** [review]

**Pull request merged**